src/Apps/Order/Routes/Shipping2/Components/ShippingQuotes2.tsx

```diff
@@ -126,3 +128,42 @@ export const shippingQuoteDisplayNames = {
   standard: "Standard",
   priority: "Priority",
 }
+
+const useAutoSelectBestShippingQuote = (
+  quotes: Array<{
+    readonly id: string
+    readonly isSelected: boolean
+    readonly price: string | null | undefined
+    readonly priceCents: number
+    readonly typeName: string
+  }>
+) => {
+  const shippingContext = useShippingContext()
+  // The best available quote is the one that is selected on the client,
+  // or the one that is already selected on the server,
+  // or the first quote in the list
+  const bestArtsyShippingQuoteId =
+    quotes.find(
+      quote => quote.id === shippingContext.state.selectedShippingQuoteId
+    )?.id ||
+    quotes.find(quote => quote.isSelected)?.id ||
+    quotes?.[0]?.id
+
+  const setSelectedShippingQuote =
+    shippingContext.actions.setSelectedShippingQuote
+  useEffect(() => {
```

Comment:
Line break between 153 and 154 (again, try really really hard to make this an automatic habit so I can stop flagging basic readability stuff in every PR 🙏 readability has historically been something that needs to improve, but you have to work with me; i'm not flagging this stuff to be pedantic)

Reply:
To be clear, here are the ground rules: _things that are different from one another_ require a line break in between them. A variable is different from a useEffect; a code comment is different from an unrelated variable; a for loop is different from a function call -- and so on and so forth. Internalize this completely! Help your comrades out 👍